modules/database/src/ioc/as/asCa.c

```diff
@@ -225,6 +225,10 @@ static void asCaTask(void)
         if(asCaDebug) printf("asCaTask has cleared all channels\n");
         epicsEventSignal(asCaTaskWait);
     }
+
+    /* ATM never reached, just a placeholder */
+
+    taskwdRemove(0);
```

Comment:
It might be clearer to add something like `cantProceed("Unreachable.  Perpetual thread.");` in addition.